lib/replace-in-file.spec.js

```diff
@@ -3,7 +3,7 @@
 /**
  * Dependencies
  */
-const replace = require('./replace-in-file');
+import replace, {sync, replaceInFile, replaceInFileSync} from './replace-in-file'
```

Comment:
I am a bit surprised that this works and passes the travis build in Node versions 8 and 10?
I thought ES6 style import were only available in Node from version 12 onwards.